.gitignore

```diff
@@ -17,3 +17,6 @@ src/Dockerrun.aws.json
 .idea
 src/api/tests/TestCases.json
 src/api/store/ZIPCODE_FIXES.json
+**/**/migrations
+**/**/settings.py
+src/_main_/settings.py
```

Comment:
settings.py is an important part of the repository.  Not to be ignored.  Just don't commit it if you don't want your changes.